.env

```diff
@@ -26,12 +26,25 @@ EXCLUDE_FROM_LB_WAIT_TIME_IN_SEC=0
 # option, which could potential exclude all nodes from load balancer at the same time.
 MAX_CONCURRENT_UPDATE=1
 
-# Update time window setting
+# Update time window setting - DEPRECATED
+# Note!! that these settings are deprecated and you should use the Scheduler settings below.
 # Brupop will operate node updates within update time window.
 # when you set up time window start and stop time, you should use UTC (24-hour time notation).
 UPDATE_WINDOW_START=0:0:0
 UPDATE_WINDOW_STOP=0:0:0
 
+# scheduler settings
+# Brupop will operate node updates on scheduled maintenance windows by using cron expressions.
+# When you set up the scheduler, you should follow cron expression rules.
+
+SCHEDULER_CRON_SECONDS=*
+SCHEDULER_CRON_MINUTES=*
+SCHEDULER_CRON_HOURS=*
+SCHEDULER_CRON_DAY_OF_MONTH=*
+SCHEDULER_CRON_MONTH=*
+SCHEDULER_CRON_DAY_OF_WEEK=*
+SCHEDULER_CRON_YEARS=*
```

Comment:
Should we consider having one Env var for the complete cron expression instead of splitting it up like this?

Reply:
I was thinking this way could help brupop to avoid invalid cron expression input at beginning. If customers miss a single space, this will exit controller with invalid cron expression. Meanwhile, I think this way can give customers a more clear place to help them to input their values.

Reply:
I think I tend to agree with @webern here. We don't do any further validation of these subfields, so we're mostly just protecting ourselves from whitespace issues. I think I'd prefer to see us take a cron expression as a single string and then use the expression parser provided in `rust-cron`.

The cost of making a mistake in this case vs the single-expression case is an error at runtime either way, so I don't think it adds much protection.